src/components/transactions/TxDetails/Summary/index.tsx

```diff
@@ -73,6 +74,16 @@ const Summary = ({ txDetails, defaultExpanded = false, hideDecodedData = false }
         </TxDataRow>
       )}
 
+      <SafenetTxChecks
+        safeTx={{
+          data: safeTxData!,
```

Comment:
Maybe, we can change `SafenetTxChecksProps` to use `Pick<SafeTransaction, "data">` so that only the `data` field is required (since that is all we use anyway).

Reply:
It will likely not work without going deeper in existing hooks and changing `useDecodeTx`